examples/Reading in linearised models.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print('#' * 20, 'SOLVE MODEL', '#'*20)
</pre><p>For print like this you can do <code>print(' SOLVE MODEL '.center(50, '#'))</code>   (the number is the total line length, including the SOLVE MODEL part</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>